### PR TITLE
docs: update contributing guides

### PR DESCRIPTION
**Pull request by levithomason**
Add more meaningful and updated contributing guides.

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -5,591 +5,105 @@ CONTRIBUTING
...
-Please follow the [Angular Git Commit Guidelines][8] format.
 
-### Commands
+### Useful Commands
 
->This list is not updated, you should run `yarn run` to see all scripts.
+>This list contains the most useful commands. You should run `yarn run` to see all scripts.
 
 ```sh
+yarn generate:component    // generates a new component. Willa sk for the name of the component
```
Comment (miroslavstastny) [inline .github/CONTRIBUTING.md]: Willa sk -> Will ask

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -5,591 +5,105 @@ CONTRIBUTING
...
-
-```js
-function  ListItem() {
-  // ...
-}
-```
+## Workflow
 
-Attach it to the parent via static properties:
+- [Setup Local Development](setup-local-development.md)
```
Comment (miroslavstastny) [inline .github/CONTRIBUTING.md:58]: Why are these links twice in the page (once here, once above Getting started)

**Comment by alinais** [comment]
Updated wording, followed suggestions in the comments and local sync. @kuzhelov , @miroslavstastny , @levithomason please have another look.

**Review by levithomason — COMMENTED** [review]
YAY 🎉

**Review by kuzhelov — APPROVED** [review]

**Pull request merged**